Cargo.toml

```diff
@@ -38,3 +38,6 @@ exclude = ["datafusion-cli"]
 [profile.release]
 codegen-units = 1
 lto = true
+
+[patch.crates-io]
```

Comment:
BTW arrow-rs 13.0.0 with these changes should be released sometime early next wee